name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/BookmarksListView.java

```diff
@@ -0,0 +1,225 @@
+package name.abuchen.portfolio.ui.views;
+
+import name.abuchen.portfolio.model.ClientSettings;
+import name.abuchen.portfolio.online.WebLocation;
+import name.abuchen.portfolio.ui.Messages;
+import name.abuchen.portfolio.ui.PortfolioPlugin;
+import name.abuchen.portfolio.ui.util.Column;
+import name.abuchen.portfolio.ui.util.ColumnEditingSupport;
+import name.abuchen.portfolio.ui.util.ColumnEditingSupport.ModificationListener;
+import name.abuchen.portfolio.ui.util.ShowHideColumnHelper;
+import name.abuchen.portfolio.ui.util.SimpleListContentProvider;
+import name.abuchen.portfolio.ui.util.StringEditingSupport;
+import name.abuchen.portfolio.ui.util.ViewerHelper;
+
+import org.eclipse.jface.action.Action;
+import org.eclipse.jface.action.ActionContributionItem;
+import org.eclipse.jface.action.IMenuListener;
+import org.eclipse.jface.action.IMenuManager;
+import org.eclipse.jface.layout.TableColumnLayout;
+import org.eclipse.jface.viewers.ColumnLabelProvider;
+import org.eclipse.jface.viewers.IStructuredSelection;
+import org.eclipse.jface.viewers.TableViewer;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.ToolBar;
+
+public class BookmarksListView extends AbstractListView implements ModificationListener
+{
+    
+    private static final String DEFAULT_LOCATION = "WebLocation"; //$NON-NLS-1$
+    private static final String DEFAULT_URL = "http://example.net/{tickerSymbol}?isin={isin}&wkn={wkn}&name={name}"; //$NON-NLS-1$
+    private TableViewer bookmarks;
+
+    @Override
+    protected String getTitle()
+    {
+        return Messages.BookmarksListView_title;
+    }
+
+    protected void addButtons(ToolBar toolBar)
+    {
+        super.addButtons(toolBar);
+        addExportButton(toolBar);        
+    }
+
+    private void addExportButton(ToolBar toolBar)
```

Comment:
Name der Methode führt etwas in die irre. Ist ja nicht ein "export button" sondern eher ein "add button".